ansible_rulebook/cli.py

```diff
@@ -172,6 +172,12 @@ def get_parser() -> argparse.ArgumentParser:
         "Default is sequential, actions will be run only after the "
         "previous one ends",
     )
+    parser.add_argument(
+        "--dev",
```

Comment:
I understand that this comes from quarkus, but I feel the term "dev" is quite generic and confusing. I suggest replace it by "hot-reload" which is more specific.

Reply:
done in 81683a9